packages/eslint-plugin-template/src/configs/accessibility.json

```diff
@@ -12,6 +12,7 @@
     "@angular-eslint/template/no-distracting-elements": "error",
     "@angular-eslint/template/role-has-required-aria": "error",
     "@angular-eslint/template/table-scope": "error",
-    "@angular-eslint/template/valid-aria": "error"
+    "@angular-eslint/template/valid-aria": "error",
+    "@angular-eslint/template/headings-consecutive-order": "error"
```

Comment:
I wonder if similar to the recommended ruleset, we should wait to add new rules to the accessibility ruleset until the next major version release. The accessibility ruleset is new to v16, but if we add a new rule on a minor version, people currently using it on their Angular 16 projects could potentially get new lint violations with a minor version update without having changed their configuration, which we should try to avoid.

If/when we do add it here, we should ensure it is alphabetized in this ruleset.

Reply:
Don't users manually add the rules they choose in the .eslintrc.json ?
If this file makes it automatically enabled rule I can remove, but I think those rules are added manually so still if people don't know about it, they won't add it (or if they don't want to).

I can alphabetize it , sure

Reply:
This accessibility ruleset is now being configured for Angular projects with the `ng add @angular-eslint/schematics` schematic, so yes, rules in this accessibility.json files will be added to projects that use the accessibility ruleset configuration, which is now included by default.